src/App/App.jsx

```
@@ -93,6 +94,10 @@
                   </li>
                 ))}
               </ul>
+              <button>
+                <img src="icon.png" />
```

ESLint

[error] img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.

(jsx-a11y/alt-text)